internal/cmd/config.go

```diff
@@ -57,6 +57,15 @@ func infraHomeDir() (string, error) {
 
 	infraDir := filepath.Join(homeDir, ".infra")
 
+	return infraDir, nil
+}
+
+func initInfraHomeDir() (string, error) {
```

Comment:
Renamed this to reflect what it actually does

Reply:
maybe to minimize changes, create a new function called `infraHomeDirName` and have `infraHomeDir` call that to get the name?